buildSrc/updateLibs.js

```diff
@@ -39,7 +40,13 @@ const clientDependencies = [
 run()
 
 async function run() {
+	// since we have git dependencies for which the version in the packages' package.json is
+	// different from the version in our package.json (the git url), we need to make sure that
+	// the installed version is in sync with the version given in package.json before we write it to
+	// vendored-versions.json
+	await $`npm i`
```

Comment:
I wouldn't add `npm i` here, that would catch me off-guard. We should take whatever is already there imo